### ISSUE TITLE
Project name might be misinterpreted 

### ISSUE THREAD

**Issue by TheOneKevin** [Anthonykung/Doorgy#3]
The name Doorgy may be misinterpreted as the slang term "doorgy" may also have unpleasant and vulgar connotations. Instead, Doorgi might be a better alternative as it can be Door + Corgi.

**Comment by Anthonykung** [comment]
Yes, we are aware of this issue however, that is after the PCB was created making the changes hard to implement. Changes will be made on 3.0.0 updates along with a new NPM listing.
